core/src/main/java/org/frankframework/receivers/JavaListener.java

```diff
@@ -74,11 +75,11 @@
  * </p>
  * @author Gerrit van Brakel
  */
+@Log4j2
+@DestinationType(Type.JVM)
```

Comment:
`Type.JVM`, of `Type.ADAPTER`?

Reply:
Deze is echt JVM. Kan via de service dispatcher binnen komen vanuit een andere app in de zelfde JVM

Reply:
Ik weet dat het kan, maar is het in de praktijk nu niet altijd een andere Adapter?

Heel veel mensen gebruiken dit om sub-adapters aan te roepen, maar bijna niemand gebruikt toch de (DLL) service dispatcher?